homeassistant/components/mvglive/sensor.py

```diff
@@ -18,44 +19,31 @@
 _LOGGER = logging.getLogger(__name__)
 
 CONF_NEXT_DEPARTURE = "nextdeparture"
-
 CONF_STATION = "station"
 CONF_DESTINATIONS = "destinations"
-CONF_DIRECTIONS = "directions"
 CONF_LINES = "lines"
 CONF_PRODUCTS = "products"
 CONF_TIMEOFFSET = "timeoffset"
 CONF_NUMBER = "number"
 
-DEFAULT_PRODUCT = ["U-Bahn", "Tram", "Bus", "ExpressBus", "S-Bahn", "Nachteule"]
+NONE_ICON = "mdi:clock"
+
+ATTRIBUTION = "Data provided by mvg.de"
 
-ICONS = {
-    "U-Bahn": "mdi:subway",
-    "Tram": "mdi:tram",
-    "Bus": "mdi:bus",
-    "ExpressBus": "mdi:bus",
-    "S-Bahn": "mdi:train",
-    "Nachteule": "mdi:owl",
-    "SEV": "mdi:checkbox-blank-circle-outline",
-    "-": "mdi:clock",
-}
-ATTRIBUTION = "Data provided by MVG-live.de"
+ENTITY_ID_FORMAT = "mvglive.{}"
```

Comment:
This is very wrong as it should be `sensor.xyz`
Not sure why you want to mod this anyhow